circleci.go

```diff
@@ -148,15 +113,15 @@ func (c *Client) BootstrapBuilds(db RootDB) error {
 					fmt.Printf(
 						"Quitting after 30 days of builds; artifacts won't exist further back.\n",
 					)
-					goto report
+					goto report // WTF
```

Comment:
lol

Reply:
cleanest way to break out of two loops.  Some modern languages have labeled loops so you can `break outer_loop;` but go is too cool for that.